### PR TITLE
dom-manipulation - Khanh Nguyen

### PR DESCRIPTION
**Pull request by khanhnguyenq**
https://github.com/khanhnguyenq/c1023-code-solutions/assets/114459075/c036c3d2-c101-40ca-a43c-b7d0c8e2e711

### CODE REVIEW

File: dom-manipulation/main.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  if (count <= 4) {
+    $hotButton.className = 'hot-button cold';
+  } else if (count <= 7) {
+    $hotButton.className = 'hot-button cool';
+  } else if (count <= 10) {
+    $hotButton.className = 'hot-button tepid';
+  } else if (count <= 13) {
+    $hotButton.className = 'hot-button warm';
+  } else if (count <= 16) {
+    $hotButton.className = 'hot-button hot';
```
Comment (shawnkost) [inline dom-manipulation/main.js]: These should just be `less than` rather than `less than or equal to`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Khanh!

File: dom-manipulation/main.js (hunk 1 above)
Comment (khanhnguyenq) [reply dom-manipulation/main.js]: Screenshot 2023-11-06 at 15 50 42
 
Thanks Shawn! repushed exercise and updated screenshot attached.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**